string id
string state
float32 pos_rel
float32 pos_abs
float32 input_pos
